packages/compartment-mapper/src/import-hook.js

```diff
@@ -68,10 +91,68 @@ const nodejsConventionSearchSuffixes = [
   '/index.node',
 ];
 
+/**
+ * Given a module specifier which is an absolute path, attempt to match it with
+ * an existing compartment; return a {@link RedirectStaticModuleInterface} if found.
```

Comment:
@kriskowal 

Neither `SourceModuleDescriptor` nor `NamespaceModuleDescriptor` is valid, in this case.  The former because we do not have a `source` property, and the latter because we do not have a `namespace` property; all we have is `{string} specifier` and `{Compartment} compartment`.

Reply:
The difference is only that the the Moddable folks preferred to rename `specifier` to`source` so that it is clear that the referent (still a string) means that this compartment adopts the other compartment’s ModuleSource, vs `namespace` when it shares the other compartment’s module instance. The name `specifier` left that ambiguous.